google-http-client-gson/src/main/java/com/google/api/client/json/gson/GsonFactory.java

```diff
@@ -90,4 +92,22 @@ public JsonGenerator createJsonGenerator(OutputStream out, Charset enc) {
   public JsonGenerator createJsonGenerator(Writer writer) {
     return new GsonGenerator(this, new JsonWriter(writer));
   }
+
+  /** Returns true if it is lenient to input JSON value. */
+  boolean getReadLeniency() {
+    return readLeniency;
+  }
+
+  private GsonFactory createCopy() {
```

Comment:
Consider `protected` since this class is not `final`.

Reply:
That's nice idea to let subclasses to define how to create a copy. Updating.